tests/launch-fastify.test.js

```diff
@@ -194,11 +194,18 @@ test('Test custom serializers', async assert => {
         assert.notOk(secondLine.res)
         assert.ok(secondLine.responseTime)
         assert.strictSame(secondLine.http, {
+          request: {
+            method: 'GET',
+          },
           response: {
             statusCode: 200,
             body: { bytes: '13' },
           },
         })
+        assert.strictSame(secondLine.url, { path: '/' })
+        assert.strictSame(secondLine.userAgent, { original: 'lightMyRequest' })
+        assert.strictSame(secondLine.host, { hostname: 'localhost:80', ip: '127.0.0.1' })
```

Comment:
Could you add a test assertion about reqId in log?

Reply:
I've added it to the assertions for both lines